x-pack/solutions/observability/plugins/slo/server/lib/rules/slo_burn_rate/executor.ts

```diff
@@ -106,13 +106,19 @@ export const getRuleExecutor = (basePath: IBasePath) =>
           ).activeRules
         : [];
 
+    const resultGroupSet = new Set<string>();
+    for (const result of results) {
+      resultGroupSet.add(result.instanceId);
+    }
+
```

Comment:
Where are we using this?

Reply:
ah, it's a left over code from previous approach of using instance Id for parsing groups. I will remove it.